<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Meue</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/tetris-icon-25.jpg</normaloff>:/image/tetris-icon-25.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{         /*类名*/
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                 stop:0 #18545A,
                                 stop:1 #F1F2B5);
    border-radius: 10px;             /*四角圆角的直径*/
    border-style:solid;             /*边框样式（实线）*/
    border-width:2px;             /*边框宽度*/
    border-color:rgb(213, 213, 213);     /*边框颜色*/
	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正像素18</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgba(0,153,255,75); /*背景色*/ 
	border: transparent;
	border-style: outset;
	border-width: 1px;
	border-radius: 10px; /*边界圆滑*/
	font: bold 18px;
	min-width:2em;
	color: black; /*字体颜色*/
	font-family: 方正像素18;
	padding: 2px;

}
 
QPushButton:hover
{
	background-color: rgba(0,141,235,75);
}
 
QPushButton:pressed
{
	background-color: rgba(0,129,214,80); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>开  始  游  戏</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="LoadButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正像素18</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgba(0,153,255,75); /*背景色*/ 
	border: transparent;
	border-style: outset;
	border-width: 1px;
	border-radius: 10px; /*边界圆滑*/
	font: bold 18px;
	min-width:2em;
	color: black; /*字体颜色*/
	font-family: 方正像素18;
	padding: 2px;

}
 
QPushButton:hover
{
	background-color: rgba(0,141,235,75);
}
 
QPushButton:pressed
{
	background-color: rgba(0,129,214,80); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>加  载  游  戏</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="SettingButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgba(0,153,255,75); /*背景色*/ 
	border: transparent;
	border-style: outset;
	border-width: 1px;
	border-radius: 10px; /*边界圆滑*/
	font: bold 18px;
	min-width:2em;
	color: black; /*字体颜色*/
	font-family: 方正像素18;
	padding: 2px;

}
 
QPushButton:hover
{
	background-color: rgba(0,141,235,75);
}
 
QPushButton:pressed
{
	background-color: rgba(0,129,214,80); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>设  置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="QuitButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgba(0,153,255,75); /*背景色*/ 
	border: transparent;
	border-style: outset;
	border-width: 1px;
	border-radius: 10px; /*边界圆滑*/
	font: bold 18px;
	min-width:2em;
	color: black; /*字体颜色*/
	font-family: 方正像素18;
	padding: 2px;

}
 
QPushButton:hover
{
	background-color: rgba(0,141,235,75);
}
 
QPushButton:pressed
{
	background-color: rgba(0,129,214,80); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>退  出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="MinimizeButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background: transparent; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border: none;
	min-width:2em;
	padding: 5px;
	image: url(:/image/minimize2.png);
}
 
QPushButton:hover
{
	background-color: rgba(0,0,0,50);
}
 
QPushButton:pressed
{
	background-color: rgba(0,0,140,75); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background: transparent; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border: none;
	min-width:2em;
	padding: 5px;
	image: url(:/image/close.png);
}
 
QPushButton:hover
{
	background-color: rgba(255,0,0,50);
}
 
QPushButton:pressed
{
	background-color: rgba(165,42,42,75); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel 
{
	
	image: url(:/image/Tetris_logo.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>作者：ngx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
